vitamin-r: change license to :commercial; add older versions for old OSs

johntrandall opened the pull request:
I'm not sure if I handled the appcast URL correctly.
What is the best way to handle it when the appcast doesn't include older versions in it?

vitorgalvao commented:
+ `appcast` should only be present in the most up-to-date one.
+ Double quotes are reserved for cases where string interpolation is needed. Please change other cases to single quotes.
+ There are too many empty lines, the cask is really confusing, please remove empty lines inside each `elsif`.
+ Those comments should be removed.
+ Prior to version `2` the app must’ve have a different name, correct? `app` needs then to be set in each, with the appropriate name.

johntrandall commented:
changed double quotes to single quotes. 
appcast now only appears once, removed extra entries (and corresponding comments)
removed empty lines as requested. 
corrected app for older, pre V2 cases

vitorgalvao commented on Casks/vitamin-r.rb:
```diff
@@ -1,12 +1,32 @@
+    url 'http://www.publicspace.net/download/Vitamin.dmg'
+    appcast 'http://www.publicspace.net/app/vitamin2.xml'
+  end
+    name 'Vitamin-R'
```
The empty line above `name` should remain.

vitorgalvao commented on Casks/vitamin-r.rb:
```diff
@@ -1,12 +1,32 @@
+    url 'http://www.publicspace.net/download/Vitamin_2_06.dmg'
+    app 'Vitamin-R 2.app'
+  elsif MacOS.release <= :mavericks
+    # last version with compatablity for both :mountain_lion and :mavericks
```
This comment should also be removed.

vitorgalvao commented:
Thank you. Looking good. Only a few more tweaks. After could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

johntrandall commented:
rebased

vitorgalvao commented:
Looks great. Thank you for the contribution and the fast responses.
